src/format/index.js

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import ReactDOM from 'react-dom'
 import _ from 'lodash'
 import moment from 'moment'
+import {Link} from 'react-router'
```

Comment:
I get errors when I do `import Link from 'react-router'` (without the curly braces).  This is how they do it in their tutorial [here](https://github.com/reactjs/react-router-tutorial/blob/start/lessons/03-navigating-with-link.md).  Other imports obviously don't need it though, what's the deal?